<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>621</width>
      <height>461</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>621</width>
      <height>461</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>621</width>
      <height>461</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:10px;</string>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Meses</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Clientes</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Calzado</string>
     </attribute>
     <widget class="QLineEdit" name="tipocalzado_lineedit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Tipo de Calzado</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>20</y>
        <width>231</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Futura Maxi CG Bold&quot;;
color: black;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Alta Calzado&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="alta_pushbutton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>390</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura Maxi CG Bold</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#alta_pushbutton{
	background-color:#c3227f;
	color: white;
	border-radius: 5px;
}

QPushButton#alta_pushbutton:pressed{
	padding-left: 5px;
	padding-top:5px;
	background-color: rgba(255, 131, 145, 255);
 	background-position: calc(100% - 10px)center;
}

QPushButton#alta_pushbutton:hover{
	background-color: #fdbb2d;
}</string>
      </property>
      <property name="text">
       <string>Alta</string>
      </property>
     </widget>
     <widget class="QLabel" name="exito_lineedit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>350</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="servicio_lineedit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>140</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Servicio Contratado</string>
      </property>
     </widget>
     <widget class="QLabel" name="aviso_lineedit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>370</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="talla_lineedit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Talla</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="color_lineedit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>190</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Color principal</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="detalles_lineedit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>90</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Detalles</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rack_lineedit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>140</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Rack</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="cliente_lineedit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Cliente</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="extra_lineedit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Extra</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="marca_lineedit">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>290</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Marca</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="materiales_lineedit">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>290</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="placeholderText">
       <string>Materiales</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Empleados</string>
     </attribute>
     <widget class="QLabel" name="aviso_lineedit_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>310</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Servicios</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>30</y>
        <width>261</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 18pt &quot;Futura Maxi CG Bold&quot;;
color: black;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Alta Empleado&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nombre_empleado_lineedit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="correo_empleado_lineedit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>150</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Correo</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="celular_empleado_lineedit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>200</y>
        <width>211</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>MS Shell Dlg 2</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(0,0,0,0);
border: 2px solid rgba(0,0,0,0);
border-bottom-color: #fdbb2d;
color: rgb(0,0,0);
padding-bottom: 7px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Celular</string>
      </property>
     </widget>
     <widget class="QPushButton" name="alta_pushbutton_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>290</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Futura Maxi CG Bold</family>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#alta_pushbutton_3{
	background-color:#c3227f;
	color: white;
	border-radius: 5px;
}

QPushButton#alta_pushbutton_3:pressed{
	padding-left: 5px;
	padding-top:5px;
	background-color: rgba(255, 131, 145, 255);
 	background-position: calc(100% - 10px)center;
}

QPushButton#alta_pushbutton_3:hover{
	background-color: #fdbb2d;
}</string>
      </property>
      <property name="text">
       <string>Alta</string>
      </property>
     </widget>
     <widget class="QLabel" name="exito_lineedit_3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: green;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="aviso_lineedit_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>260</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Otros</string>
     </attribute>
     <widget class="QLabel" name="aviso_lineedit_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>410</y>
        <width>191</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: red;</string>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>31</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
background-color:qlineargradient(spread:pad, x1:1, y1:2, x2:0, y2:0, stop:0 rgba(195,34,127), stop:0.502488 rgba(195,34,127,1), stop:1 rgba(253,187,45,1));


border-radius: 15px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="salirButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #fdbb2d;</string>
    </property>
    <property name="text">
     <string>❌​</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
